### DISCUSSION TITLE
Network optimization

### DISCUSSION THREAD

**Discussion by GentleEnvy** [FrameMuse/react-template#53] [Ideas]
- compressing via gzip
- supports HTTP2

**Comment by FrameMuse** [comment]
You should describe your point of view better since it is not clear what you want and why

**Reply by FrameMuse** [reply]
You could say more about HTTP2: why you want it to be supported, come up with pros and cons

**Reply by FrameMuse** [reply]
You might clarify `gzip` and provide some comparisons

**Reply by GentleEnvy** [reply]
HTTP 2 speeds up downloads and makes optimal use of TCP connections

**Reply by GentleEnvy** [reply]
I recommend using [Zopfli](https://en.wikipedia.org/wiki/Zopfli) because it has "higher compression ratios than mainstream Deflate". This is well suited for compressing the static files

**Comment by FrameMuse** [comment]
What do you mean by "Network optimization"? Is it optimizing an API or user network usage?

**Reply by GentleEnvy** [reply]
Both if possible
